posthog/migrations/0256_pluginstorage_timestamp.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.14 on 2022-08-16 15:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    atomic = False
+
+    dependencies = [
+        ("posthog", "0255_user_prompt_sequence_state"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(model_name="pluginstorage", name="posthog_unique_plugin_storage_key",),
```

Comment:
Old code will be running during this migration. What happens to old upsert code between this and the unique constraint being added below?